### PR TITLE
Add OWNERS and pr template for mungegithub

### PR DESCRIPTION
**Pull request by yutongz**
Add required files for Mungegithub.

OWNERS file defines who can approve prs in this repo and who are recommended reviewer.
PR template helps people add release-note when creating prs. Simply add release-note or "NONE" between ```, then mungegithub will add proper release-note label. More details at https://github.com/istio/istio/pull/525

More details about [Mungegithub](https://github.com/istio/test-infra/tree/master/mungegithub/deployment)

---
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/1024)

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+reviewers:
+  - geeknoid
+  - mandarjog
+  - Douglas-reid
+  - ZackButcher
+  - sebastienvas
+approvers:
```
Comment (ZackButcher) [inline OWNERS]: We need more approvers than just mtail and mjog. @geeknoid I don't see a reason all of the current core contributers can't be approvers.

Comment (geeknoid) [inline OWNERS]: Yeah, let's add ZackButcher, Douglas-Reid, and guptasu

Comment (yutongz) [inline OWNERS]: Agree, felt like we need more, just don't know who. Added. PTAL

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,12 @@
+reviewers:
```
Comment (ZackButcher) [inline OWNERS]: nit: alphabetical order in both lists would be nice.

File: OWNERS (hunk 1 above)
Comment (guptasu) [inline OWNERS]: What is this list used for ?

Comment (ZackButcher) [inline OWNERS]: https://github.com/kubernetes/test-infra/tree/master/mungegithub/mungers/approvers

File: OWNERS (hunk 2 above)
Comment (yutongz) [inline OWNERS]: Absolutely

**Comment by yutongz** [comment]
Can anyone help me merge this?

**Pull request merged**